scripts/Plotter.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import json
+import os
+import re
+from dataclasses import dataclass
+
+import click
+import matplotlib.pyplot as plt
+
+
+@dataclass
+class BenchResults:
```

Comment:
This way of storing/processing results makes it difficult to add new benchmarks. Should be general data structure (list etc.), enabling to add new benchmarks at subsequent indices and process them in loops (not fixed to 4 particular tests)

Reply:
Done